src/Editable.php

```diff
@@ -96,9 +96,7 @@ public function init()
             throw new InvalidConfigException("'Url' property must be specified.");
         }
         if (!isset($this->options['id'])) {
-            $this->options['id'] = $this->hasModel() && empty($this->id)
-                ? Html::getInputId($this->model, $this->attribute)
-                : $this->getId();
```

Comment:
It was added in order to remove the error in js. When on the page simultaneously there are several similar cards, for example, user profile description, then email field will be displayed in each card with the same `id="user-emai"`. The result is `Editable` works only for the first card.